marimo/_cli/cli.py

```diff
@@ -345,6 +345,7 @@ def _get_stdin_contents() -> str | None:
     type=bool,
     help="Watch the file for changes and reload the code when saved in another editor.",
 )
+@click.option("--auto-download", default=None, type=str, hidden=True)
```

Comment:
can we make this a list, `multiple=True`, so we can do `--auto-download ipynb --auto-download html`